### PR TITLE
PR workflow diagram update

### PR DESCRIPTION
**Pull request by raych1**
The action suggested for PR authors is not correct in the diagram. 

The SDK breaking change will be evaluated within two business days after the completion of the first two steps. SDK breaking change review is conducted proactively and doesn’t need PR authors to post a message in the support channel. Reviewers will reserve some time every workday to review the candidate PRs.  

@konrad-jamrozik @weshaggard please review and merge this PR, thanks.

### CODE REVIEW

**Comment by weshaggard** [comment]
@raych1 if they don't get an answer or have questions where should they direct them? Also is there a query or some way to see them on the list of things to be reviewed?

Also for my information is the plan to centrally triage and review these?

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Please understand this diagram before proceeding. It explains how to get your PR approved & merged.
 
-![diagram](https://github.com/Azure/azure-rest-api-specs/assets/4429827/029a11fd-eaf9-4865-9d77-a16b5ea5eb26)
+![diagram](https://github.com/Azure/azure-rest-api-specs/assets/20296335/93ad7897-061a-4724-8a51-dcb797d5a6e0)
```
Comment (konrad-jamrozik) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:7]: Hey I see that in the new diagram there are red squiggles / underlines under some label names. I think it makes the diagram a little bit harder to read and would be nice to remove. But if you don't want to do it now, let's just remember to take care of this on next diagram update.

**Review by konrad-jamrozik — APPROVED** [review]
Approved and left few comments.

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md [hunk 2]
```diff
@@ -39,10 +39,8 @@ If the automation determines you have breaking changes, i.e. Step 1 from the dia
 [1] [ARM review queue] (for **merge** queues, see [4])  
 The PRs are processed by time opened, ascending. Your PR may show up on 2nd or later page. 
 If you addressed Step 1 from the diagram and your PR is not showing up in the queue, ensure the label `ARMChangesRequested` 
-is removed from your PR. This should cause the label `WaitForARMFeedback` to be added.  
-[2] https://aka.ms/azsdk/support/specreview-channel  
-[3] [List of SDK breaking changes approvers] in pinned Teams announcement
```
Comment (konrad-jamrozik) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:44]: Ray, is the list of SDK breaking changes approved still correct? Should we continue maintaining it as a pinned Teams announcement or remove it / migrate to https://aka.ms/azsdkdocs ?

Comment (konrad-jamrozik) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:44]: I think you should also remove the definition of `[List of SDK breaking changes approvers]` from the end of this file.

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md (hunk 1 above)
Comment (raych1) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:7]: Removed.

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md (hunk 2 above)
Comment (raych1) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:44]: The pinned post is still valid. We shall update it next time when we roll out the new SDK breaking change review workflow.

Comment (raych1) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:44]: Sure, I removed it.

**Comment by raych1** [comment]
> @raych1 if they don't get an answer or have questions where should they direct them? Also is there a query or some way to see them on the list of things to be reviewed?
> 
> Also for my information is the plan to centrally triage and review these?

@weshaggard , the pinned post for SDK reviewers in Teams channel is still active and available for users to seek further assistance.  The purpose of this PR is to fix the issue in the diagram.

As for a direct GitHub query link for the candidate PRs requiring SDK breaking change review, I haven't figured out yet. The challenge is that the query needs to cover various scenarios, such as PRs that do or don't require API breaking change review, and those that do or don't require ARM review.

At this time, I don't foresee a plan to centralize the SDK breaking change review process in near future. However, we will explore when we roll out the new review workflow.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**